.github/workflows/cypress-testing.yml

```diff
@@ -6,11 +6,11 @@ on:
       - apps/**
       - packages/**
   pull_request:
-    branches: ['v3', 'v3*']
-    types: [opened, synchronize, reopened]
-    paths:
-      - apps/**
-      - packages/**
+    # branches: ['v3', 'v3*']
+    # types: [opened, synchronize, reopened]
+    # paths:
+    #   - apps/**
+    #   - packages/**
```

Comment:
We have to re-introduce the constraints where cypress is run before merging this branch